Forge/Shared/Data/CharacterTagModel.cs

```diff
@@ -22,5 +22,7 @@ public override bool Equals(Object obj)
                 return Id == t.Id;
             }
         }
+
+        public bool IsDeleted { get; set; }
```

Comment:
Please place it in an appropriate place (with other properties). Formatting is important.